_blog/2018/soft-multitenancy.md

```diff
@@ -0,0 +1,316 @@
+https://github.com/kubernetes/community/blob/master/wg-multitenancy/README.md)
+is working to define the multitenant use cases and functionality that should be available
+within Kubernetes. However, from their work so far it is clear that only "soft multitenancy"
+is possible due to the inability to fully protect against malicious containers or workloads
+gaining access to other tenant's pods or kernel resources.
+
+## Soft multitenancy
+For this blog, "soft multitenancy" is defined as having a single Kubernetes control plane
+with multiple Istio control planes and multiple meshes, one control plane and one mesh
+per tenant. The cluster administrator gets control and visibility across all the Istio
+control planes, while the tenant administrator only gets control of a specific Istio
+instance. Separation between the tenants is provided by Kubernetes namespaces and RBAC.
+
+One use case for this deployment model is a shared corporate infrastructure where malicious
+actions are not expected, but a clean separation of the tenants is still required.
+
+Potential future Istio multitenant deployment models are described at the bottom of this
+blog.
+
+**Note:** This blog is a high-level description of how to deploy Istio in a
+limited multitenancy environment. The [docs]({{home}}/docs/) section will be updated
+when official multitenancy support is provided.
+
+## Deployment
+### Multiple Istio control planes
+Deploying multiple Istio control planes starts by replacing all `namespace` references
+in a manifest file with the desired namespace. Using istio.yaml as an example, if two tenant
+level Istio control planes are required; the first can use the istio.yaml default name of
+*istio-system* and a second control plane can be created by generating a new yaml file with
+a different namespace.
+```bash
+cat istio.yaml | sed s/istio-system/istio-system1/g > istio-system1.yaml
+```
+Note that the execution of these two yaml files is the responsibility of the cluster
+administrator, not the tenant level adminstrator. Additional RBAC restrictions will also
+need to be configured and applied by the cluster administrator, limiting the tenant
+administrator to only the assigned namespace.
+
+If the Istio [addons]({{home}}/docs/tasks/telemetry/) are required then the manifests must
+be updated to match the configured `namespace` in use by the tenant's Istio control plane.
+
+### Split common and namespace specific resources
+The manifest files in the Istio repositories create both common resources that would
+be used by all Istio control planes as well as resources that are replicated per control
+plane. Although it is a simple matter to deploy multiple control planes by replacing the
+*istio-system* namespace references as described above, a better approach is to split the
+manifests into a common part that is deployed once for all tenants and a tenant
+specific portion. All the CustomResourceDefinitions (CRDs), the roles and the role
```

Comment:
portion -> part, to match previous use in sentence.

Reply:
Done